src/main/java/de/retest/recheck/ui/diff/Alignment.java

```diff
@@ -23,7 +25,8 @@
 	private final Map<Element, Element> alignment;
 	private final Map<Element, Element> expectedMapOfElementTree = new HashMap<>();
 	private final Map<Element, Element> actualMapOfElementTree = new HashMap<>();
-	private final Double minimumMatch = getRequiredMinimumMatch();
+	private final Double minimumMatch =
+			getConfiguredDouble( ELEMENT_MATCH_THRESHOLD_PROPERTY, ELEMENT_MATCH_THRESHOLD_DEFAULT );
```

Comment:
I would inline this so that it is evaluated every time it is needed (in case it is not set at construction of this class). Also this is only used once.

Reply:
I don't think this is necessary.